clang/test/Driver/arm-alignment.c

```diff
@@ -22,6 +22,14 @@
 // RUN: %clang -target armv7-windows -### %s 2> %t
 // RUN: FileCheck --check-prefix=CHECK-UNALIGNED-ARM < %t %s
 
+/// Ensure that by default before ARMv7 we default to +strict-align
+// RUN: %clang -target armv6 -### %s 2> %t
```

Comment:
`--target=` for new tests.

`-target ` has been deprecated since about Clang 3.4